src/components/Status.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+class Status extends React.Component {
+  static propTypes = {
+    message: PropTypes.string,
+    type: PropTypes.string
+  }
+  render() {
+    return (
+      <section className={`status ${this.props.type}`}>
+      {this.props.message}
+      </section>
```

Comment:
You've got classes defined for this `Status` component, but no CSS that uses them!